backend/src/handlers/events_attendance_handler.go

```diff
@@ -103,6 +103,13 @@ func (srv *Server) handleAddAttendanceForEvent(w http.ResponseWriter, r *http.Re
 	if err := srv.Db.LogUserAttendance(attendances); err != nil {
 		return newDatabaseServiceError(err)
 	}
+
+	for _, att := range attendances {
+		if err := srv.Db.CreateAttendanceAuditTrail(r.Context(), &att, &args.UserID, class.Name); err != nil {
+			log.error("failed to create attendance audit trail: " + err.Error())
+		}
+	}
```

Comment:
@CK-7vn Since this "auditing" I'm not sure how it compares in mission-criticality as logging the attendance. I suppose one could argume it's the same.

But the reason I ask is should the Audit Trail be in the same transaction as the logging of attendance? Otherwise I suppose the audit could fail and we have no record of the actual attendance being recorded.